src/main/java/it/pagopa/ecommerce/commons/utils/RedirectConfigurationKeysConfig.java

```diff
@@ -0,0 +1,39 @@
+package it.pagopa.ecommerce.commons.utils;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class RedirectConfigurationKeysConfig {
+
+    private final Map<String, URI> redirectBeApiCallUriMap;
+
+    public RedirectConfigurationKeysConfig(Map<String, URI> redirectBeApiCallUriMap) {
```

Comment:
add input Set of expected keys to be present into map and move here the check for all existing keys to be present as done [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/6fd2d721add5a0b66a1bf5550d605b2f8b2eff17/src/main/java/it/pagopa/transactions/configurations/RedirectConfigurationsBuilder.java#L52-L62)

Reply:
Done.